service/history/replication/stream_receiver.go

```diff
@@ -262,15 +262,23 @@ func (r *StreamReceiverImpl) ackMessage(
 			r.logger.Warn("Tiered stack mode. Have to wait for both high and low priority tracker received at least one batch of tasks before acking.")
 			return 0, nil
 		}
+		highPriorityFlowControlCommand := r.flowController.GetFlowControlInfo(enums.TASK_PRIORITY_HIGH)
+		if highPriorityFlowControlCommand == enums.REPLICATION_FLOW_CONTROL_COMMAND_PAUSE {
```

Comment:
Possible to log when resuming the flow?

Reply:
It's possible, but I don't think it is necessary. The sync status was invoked in 1 second interval. When we stop seeing the pausing log, that indicating it is resuming. Unless you want to see additional info?